README.rst

```diff
@@ -38,25 +38,29 @@ Quick start
 
 A common CLI workflow includes:
 
-+-----------------------------------+----------------------------------------------------------------------------+
-| Task                              | Terminal                                                                   |
-+===================================+============================================================================+
-| Track data                        | | ``$ git add train.py``                                                   |
-|                                   | | ``$ dvc add images.zip``                                                 |
-+-----------------------------------+----------------------------------------------------------------------------+
-| Connect code and data             | | ``$ dvc run -n prepare -d images.zip -o images/ unzip -q images.zip``    |
-|                                   | | ``$ dvc run -n train -d images/ -d train.py -o model.p python train.py`` |
-+-----------------------------------+----------------------------------------------------------------------------+
-| Make changes and reproduce        | | ``$ vi train.py``                                                        |
-|                                   | | ``$ dvc repro model.p.dvc``                                              |
-+-----------------------------------+----------------------------------------------------------------------------+
-| Share code                        | | ``$ git add .``                                                          |
-|                                   | | ``$ git commit -m 'The baseline model'``                                 |
-|                                   | | ``$ git push``                                                           |
-+-----------------------------------+----------------------------------------------------------------------------+
-| Share data and ML models          | | ``$ dvc remote add myremote -d s3://mybucket/image_cnn``                 |
-|                                   | | ``$ dvc push``                                                           |
-+-----------------------------------+----------------------------------------------------------------------------+
+
++-----------------------------------+----------------------------------------------------------------------------------+
+| Task                              | Terminal                                                                         |
++===================================+==================================================================================+
+| Track data                        | | ``$ git add train.py params.yaml``                                             |
+|                                   | | ``$ dvc add images.zip``                                                       |
```

Comment:
remove `.zip` from images I think. make it `dvc add images/`

Reply:
I thought the point was to show a multi-step pipeline in the next step. Do you think we should get rid of that, or make up some new transformation as the first step?

Reply:
I see, yes, let's keep ... this is not good though by itself (bad pattern for DVC to use archives)

Reply:
Made a new transformation